mentat/feature_filters/llm_feature_filter.py

```diff
@@ -86,11 +89,15 @@ async def filter(
             messages.append(
                 {"role": "system", "content": f"Expected Edits:\n{self.expected_edits}"}
             )
+
+        self.report_loading("Asking LLM to filter out irrelevant context...", 80)
         message = await call_llm_api_sync(model, messages)
+
+        self.report_loading("Parsing LLM response...", 10)
         try:
             selected_refs = json.loads(message)  # type: ignore
         except json.JSONDecodeError:
-            raise ValueError(f"The response is not valid json: {message}")
+            raise ModelError(f"The response is not valid json: {message}")
```

Comment:
In my testing I hit this exception a lot.

Reply:
Per your other PR and the conversation there, I'm going to switch off `use_llm` which will bypass this code, and we'll rehaul it in a future PR.

Reply:
I think this could be a great use for the json format mode in the future!